content/en/security/application_security/enabling/serverless.md

```diff
@@ -22,21 +21,25 @@ further_reading:
       text: "Application Threat Monitoring and Protection"
 ---
 
-<div class="alert alert-info">ASM support for AWS Lambda is in beta. Threat detection is done by using the Lambda extension.</div>
+{{< partial name="security-platform/appsec-serverless.html" >}}</br>
 
-You can monitor your functions running in AWS Lambda with Datadog Application Security Management (ASM). See [Compatibility][4] for information about what ASM features are supported for serverless functions.
+See [Compatibility][4] requirements for information about what ASM features are available for serverless functions.
 
-In general, setting up ASM for AWS Lambda involves:
+## AWS Lambda
+
+Configuring ASM for AWS Lambda generally involves:
 
 1. Identifying functions that are vulnerable or are under attack, which would most benefit from ASM. Find them on [the Security tab of your Service Catalog][1].
```

Comment:
I can't find this link easily, but can we replace this hyperlink url to point to this instead: https://app.datadoghq.com/services?query=type%3Afunction&env=prod&groupBy=&hostGroup=%2A&lens=Security&sort=-attackExposure&view=list